### PR TITLE
Feat/Add two feature toggles

### PR DESCRIPTION
**Pull request by ishiko732**
In TS-FSRS, there is no option to modify `enable_short_term` and `enable_fuzz`. Add two toggles to enable these features.

image

image

image

### CODE REVIEW

**Comment by Newdea** [comment]
thanks your PR, LGTM, i will merge it soon.

File: .github/workflows/e2e.yml
Comment (Newdea) [inline .github/workflows/e2e.yml:1]: hi, i noticed you've changed some of the ci/cd files, will them affect ts-fsrs?
if not, i hope just keep them as they are.

Comment (ishiko732) [reply .github/workflows/e2e.yml:1]: I will revert the commit to the version tagged as "fix lint" (3b6e3e2)

**Comment by ishiko732** [comment]
On my computer, the tests run correctly, and the coverage is 100%...
image

**Pull request merged**